src/main/java/com/minecolonies/coremod/entity/pathfinding/AbstractPathJob.java

```diff
@@ -246,8 +246,8 @@ private static boolean onALadder(@NotNull final Node node, @Nullable final Node
     private static int computeNodeKey(@NotNull final BlockPos pos)
     {
         return ((pos.getX() & 0xFFF) << 20)
-                | ((pos.getY() & 0xFF) << 12)
-                | (pos.getZ() & 0xFFF);
+                 | ((pos.getY() & 0xFF) << 12)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 12 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)